setup.py

```diff
@@ -26,6 +27,11 @@
     'backports.shutil_get_terminal_size;python_version<"3.3"'
 ]
 
+# support for win32api on Windows (win32 and win-amd64)
+# pypiwin32 is not available for python 3.4 and below
+if get_platform().startswith("win"):
```

Comment:
the windows wheel might miss this dependency
similar to https://github.com/streamlink/streamlink/pull/1758/files#diff-2eeaed663bd0d25b7e608891384b7298

Reply:
Sorry but I'm not really sure what I'm supposed to do to fix that.

Reply:
add it directly into **deps**, without `get_platform().startswith("win")`

similar to

https://github.com/streamlink/streamlink/blob/56253563e92732e2f0ebffebabc566c3bc4a2124/setup.py#L22-L24

---

also add it into `makeinstaller.sh`

https://github.com/streamlink/streamlink/blob/master/script/makeinstaller.sh#L70-L100

Reply:
> add it directly into deps, without get_platform().startswith("win")

But is it possible to check for the `sysconfig.get_platform()` with that method for the conditional dependency? The problem I mentioned in my original post was that the install would fail on msys2 because all the other methods for checking the platform have the same result as a "normal" Windows python installation. However it would then try to get pypiwin32 from pip for mingw which is not available.